src/components/10-atoms/checkbox/index.scss

```diff
@@ -32,6 +33,19 @@ axa-checkbox {
         background-color: $color-axa-blue;
       }
 
+      &:checked + .a-checkbox__icon--tick{
+        .a-checkbox__icon-tick {
+          height: 18px;
+          display: block;
+          color: $color-prim-white;;
+          fill: currentColor;
+          text-align: center;
+          padding-top: 1px;
```

Comment:
Why exact 1px? Did you calculate it or just tried it what looks best?

Reply:
I just tried this

Reply:
` text-align: center;` did not exactly positioned it in the center. And other methodes didn't work

Reply:
I guess the image has emtpy space at the bottom or something like that. So it would be cleaner to change the image. the 1px is a bit weird but its just 1px... so minor. like u want.

Reply:
I mad an issue #1595